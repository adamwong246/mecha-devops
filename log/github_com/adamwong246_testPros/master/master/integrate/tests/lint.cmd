run,--rm=true,github_com-adamwong246_testpros/master:master,npm,run,lint