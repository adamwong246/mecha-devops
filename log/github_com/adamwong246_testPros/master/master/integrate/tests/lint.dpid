7f2527d8e5b9fc37aefa3afcbcc82f6eb7ad162da1e86e7bccc7f1e0bb9a2d07