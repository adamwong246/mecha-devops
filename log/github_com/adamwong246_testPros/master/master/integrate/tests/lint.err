npm info it worked if it ends with ok
npm info using npm@3.10.10
npm info using node@v6.10.0
npm info lifecycle test-proj@1.0.0~prelint: test-proj@1.0.0
npm info lifecycle test-proj@1.0.0~lint: test-proj@1.0.0
npm info lifecycle test-proj@1.0.0~postlint: test-proj@1.0.0
npm info ok 
