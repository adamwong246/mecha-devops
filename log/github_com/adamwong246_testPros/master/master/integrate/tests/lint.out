54674c02f39fb518748b9cef0270cf5c08f99fd23d4ff57cd4a65cd6e834f509
